partiql-logical/src/lib.rs

```diff
@@ -139,6 +139,10 @@ where
         &self.edges
     }
 
+    pub fn operator(&self, id: OpId) -> Option<&T> {
+        self.nodes.get(id.0)
```

Comment:
This came about when I was testing some queries with JOINs using the AST to logical plan conversion. Don't think this is entirely correct. The `OpId`s are 1-indexed as far as I can tell. (Can look at how `add_operator(...)` which uses the number of operators https://github.com/partiql/partiql-lang-rust/blob/d3a4b46a55274368653166fed9555ab37ed36cab/partiql-logical/src/lib.rs#L74-L78). While the `nodes` list is a vec that is 0-indexed. I guess a simple fix could be to subtract 1:

```suggestion
        self.nodes.get(id.0 - 1)
```

Alternatively, could change `OpId` to be 0-indexed when adding to the logical plan (not sure how much refactoring would be required though).